.github/workflows/main.yml

```diff
@@ -55,6 +55,6 @@ jobs:
           python-version: ${{ matrix.python-version }}
           architecture: x64
       - run: |
-          pip install pytest coverage pytest-cov ${{ matrix.django-version }}
+          pip install pytest coverage pytest-cov mock ${{ matrix.django-version }}
```

Comment:
You don't need to install mock; you can use [unittest.mock](https://docs.python.org/dev/library/unittest.mock.html).